aten/src/ATen/cudnn/Descriptors.h

```diff
@@ -318,6 +318,15 @@ struct TORCH_CUDA_CPP_API RNNDescriptor : public Descriptor<
       }
     }
 #else
+    cudaDeviceProp* prop = at::cuda::getCurrentDeviceProperties();
+    auto math_type = CUDNN_DEFAULT_MATH;
+    if (prop->major >= 7) {
+      if (input_type == CUDNN_DATA_HALF) {
+        math_type = CUDNN_TENSOR_OP_MATH;
+      } else if (allow_tf32) {
```

Comment:
Hi, This is from cudnn API doc, and " With NVIDIA Ampere architecture and CUDA toolkit 11, CUDNN_DEFAULT_MATH permits TF32 Tensor Core operation and CUDNN_FMA_MATH does not."
So when allow_tf32 == false, set math_type to CUDNN_FMA_MATH?

Reply:
Nice catch! Yes that's a typo, it should be `!allow_tf32`